### DISCUSSION TITLE
Export tones to hardware

### DISCUSSION THREAD

**Discussion by fefillo** [soundshed/soundshed-app#27] [Ideas]
Is it possible to save tones from Soundshed to the 1-2-3-4 presets stored in the amp? I couldn't find a way to do it like we can with the mobile app (save to hardware preset). 

Building up on that... I think it would be even better if we could save multiple tones at once, and/or save "sets" of tones, maybe like having folders in our My Tones or something.  

A possible example of how I would use them - Take for instance Pink Floyd's Comfortably Numb. There's some good Lead, Solo #1 and Solo #2 tones out there in the community. It would be great if I could load them all up into tones 1-2-3 and then use my (soon to arrive) Bluetooth pedal to select them as I play. Then load up another set with one click for another song, style, mood or whatever.

**Comment by webprofusion-chrisc** [comment]
The soundshed app doesn't currently have the store to hardware function (I just haven't got round to it) but if you are playing any preset you can just long-press the hardware button to store it at that preset.

There is a general idea still to be implemented around 'set lists' e.g. custom groups of presets arrange either into individual songs or any other grouping you like. Your extended idea around that of applying 4 of them to the 1-4 presets in one go sounds good.
